src/components/chat/chat-comment-thread.ts

```diff
@@ -29,7 +30,7 @@ export function makeChatThreads (
         tile = content?.getTile(comment.tileId);
       }
       chatThreads.push({
-        title: isTileComment && tile ? tile.computedTitle : null,
+        title: isTileComment ? (tile ? tile.computedTitle: null) : docTitle || null,
```

Comment:
This might be more readable as:

`title: (isTileComment ? tile?.computedTitle : docTitle) ?? null`